testsweeper.hh

```diff
@@ -17,6 +17,7 @@
 #include <complex>
 #include <type_traits>
 #include <numeric>
+#include <stdint.h>
```

Comment:
Minor, but I would put this with the other C headers above, rather than with the C++ headers here.